gulpfile.js

```diff
@@ -1,19 +1,20 @@
 'use strict';
 
 var gulp = require('gulp'),
-    webserver = require('gulp-webserver'),
+    connect = require('gulp-connect'),
     gulpConfig = require('./.gulp/gulpConfig');
 
 gulp.task('test', require('./.gulp/tasks/tests')(gulp, gulpConfig));
 
 gulp.task("server", function () {
+	connect.server({
+		root: 'examples/web/',
+		port: 8000,
+		livereload: true
+  	});
+
     return gulp.src(['examples/web', 'packages/desktopjs/dist', 'packages/desktopjs-electron/dist', 'packages/desktopjs-openfin/dist'])
```

Comment:
@naknode These other paths don't seem to be served so I get 404 errors in the examples.

Reply:
Try moving the array to be the value for root and remove the return

Reply:
That seems to have worked as I can successfully navigate around the application.

In my terminal:

```bash
[11:28:44] Using gulpfile ~/code/oss/desktopJS/gulpfile.js
[11:28:44] Starting 'server'...
[11:28:44] Starting server...
[11:28:44] Server started http://localhost:8000
[11:28:44] LiveReload started on port 35729
[11:28:44] Running server
```

Please let me know what command you are using to generate those errors. I do realize that when I do `npm run build`, I get `TypeScript: 503 semantic errors`

Reply:
I just went opened localhost:8000 and nothing worked so I looked at dev tools and saw all the 404

Reply:
Even now? I just did a fresh install of my @naknode/desktopJS and I was able to get localhost:8000 up and running with gulp-connect.

Reply:
It's all good, sorry for confusion.  I thought you were asking how I knew it wasn't working.  Only thing left is to update each commit to indicate dco per contributing guide (I suggest squashing to make that easier).